iib/workers/tasks/utils.py

```diff
@@ -695,6 +696,9 @@ def wrapper(*args, **kwargs):
         finally:
             if request_log_handler:
                 logger.removeHandler(request_log_handler)
+                request_log_handler.flush()
```

Comment:
should not we flush before removing the handler? :thinking:

Reply:
An argument for doing it after is that we have a better guarantee that nothing else will be written to the handler after flushing it. Since these calls are done right next to each other, it probably doesn't make a difference.